package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/react": "^9.3.1",
```

Comment:
it's not a big deal right now, but for deployed projects it's best to install testing dependencies as a devDependency.  Use the flag `--save-dev` to save directly to dev dependencies